lockfile.json

```
@@ -240,7 +240,7 @@
     "revision": "fcc5f6f4d194dede4e676834ff28a506e39e17b4"
   },
   "tlaplus": {
-    "revision": "26bbaf5d07356cacbf21defa9316988bc732c8b3"
+    "revision": "6b82b26a742964bd93d022db134a4ac3a1a15a96"
```

I think this should be

```suggestion
    "revision": "ffa98bbbd80e425f07e92bf511e64c8b0dffc1c7"
```

---

Indeed it should.